source/cloud-native-platform/path-to-live/shutter.html.md.erb

```diff
@@ -1,54 +1,105 @@
+4. Lets assume this application need to be shuttered `www.decree-absolute.apply-divorce.service.gov.uk`
+5. Find the relevant file relating to the DNS zone `apply-divorce-service-gov-uk` (and the environment being shuttered) and make sure there is a [record](https://github.com/hmcts/azure-public-dns/blob/a24128ffdc47687937cbab37aaa46040786c7955/environments/prod/apply-divorce-service-gov-uk.yml#L41) for the service
+
+6. Now check that an equivalent shuttering [configuration](https://github.com/hmcts/azure-public-dns/blob/27f3b5df3a1d3b847f2ca196e77a41c9ab896cc1/shuttering/prod/apply-divorce-service-gov-uk.yml#L20) exists for this record
+
+- <details>
+    <summary>Adding a missing shutter config</summary>
+    <ul style="padding-left:20px">
+      <li>If no shutter config exists you will need to add it.</li>
+      <li>The shuttering file names <b>match</b> the environment file names so make sure to add it to the correct file. In this example we would need to add the shutter config value to <code>shuttering/prod/apply-divorce-service-gov-uk.yml</code></li>
+      <li>When adding a new shutter value, the <code>name</code> field must <b>match</b> that of the DNS record, this is how we know which shutter config matches which DNS record</li>
+      <div class="highlight"><pre class="highlight plaintext" tabindex="0"><code>- name: "www.decree-absolute"
+      shutter: true</code></pre></div>
+      </ul>
+    </details>
+
+7. If the shutter config exists, changing the shutter value from false to true is all that is required to enable shuttering for this service.
+
+- <details>
+    <summary>What happens when you change the shutter value</summary>
+    <p>Setting <code>shutter: true</code> will enable the shutter page and redirect all the traffic to Azure CDN.<br>
+    Setting <code>shutter: false</code> will disable the shutter page and redirect all the traffic back to Azure Frontdoor.</p>
+    </details>
+
+8. Create a PR with the changes and have it reviewed. Once merged this will update the exist record to point to the Azure Static Web App endpoint.
+
+This is a self service request and does not require Platform Operations to create, approve or merge. Therefore this can be done out of hours without having Platform Operations around for shuttering only.
+
+## Record Types
+
+There are two record types that allow shuttering to be enabled.
+These are:
+
+- A records
+  - The "A" stands for "address" and this is the most fundamental type of DNS record. It indicates the IPv4 address of a given domain.
+- CNAME records
+  - A "canonical name" (CNAME) record points from an alias domain to a "canonical" domain. A CNAME record is used in lieu of an A record, when a domain or subdomain is an alias of another domain. All CNAME records must point to a domain, never to an IP address.
+
+### Shuttering A record types
```

Comment:
Should there also be something small in this section about _how_ to shutter? Maybe just pointing to above section and some rewording?

CNAME bit has this for instance, but A records seems more like info & pre-reqs
 ```
CNAME records are much easier to shutter, for each record an equivalent shutter config must exist, as detailed in the Enable shuttering with a DNS swap section.

Simply enable or disable the shutter using True/False values
```